src/main/webapp/tests.html

```diff
@@ -13,6 +13,7 @@
   <div id="qunit-fixture">
     <header></header>
     <div id="menu"></div>
+    <textarea id="video-url" value="gjKpAAezWQA"></textarea>
```

Comment:
Is it video id?

Reply:
changed the value to a url to be consistent